modin/pandas/test/test_dataframe.py

```diff
@@ -3673,22 +3673,15 @@ def test_fillna_sanity(self):
         modin_df = pd.DataFrame(frame_data).fillna(value={"Date": df["Date2"]})
         df_equals(modin_df, result)
 
-        # TODO: Use this when Arrow issue resolves:
-        # (https://issues.apache.org/jira/browse/ARROW-2122)
-        # with timezone
-        """
-        frame_data = {'A': [pandas.Timestamp('2012-11-11 00:00:00+01:00'),
-                            pandas.NaT]}
+        frame_data = {"A": [pandas.Timestamp("2012-11-11 00:00:00+01:00"), pandas.NaT]}
         df = pandas.DataFrame(frame_data)
         modin_df = pd.DataFrame(frame_data)
-        df_equals(modin_df.fillna(method='pad'), df.fillna(method='pad'))
+        df_equals(modin_df.fillna(method="pad"), df.fillna(method="pad"))
 
-        frame_data = {'A': [pandas.NaT,
-                            pandas.Timestamp('2012-11-11 00:00:00+01:00')]}
+        frame_data = {"A": [pandas.NaT, pandas.Timestamp("2012-11-11 00:00:00+01:00")]}
         df = pandas.DataFrame(frame_data)
-        modin_df = pd.DataFrame(frame_data).fillna(method='bfill')
-        df_equals(modin_df, df.fillna(method='bfill'))
-        """
+        modin_df = pd.DataFrame(frame_data).fillna(method="bfill")
+        df_equals(modin_df, df.fillna(method="bfill"))
```

Comment:
`test_fillna_sanity` looks heavy to read, may we add `frame_data`, `method` and `value` to a `pytest.parametrize`, and then in a test function just use `eval_general`
```python
eval_general(
    modin_df,
    pandas_df,
    operation=lambda df, **kwargs: df.fillna(**kwargs),
    method=method,
    value=value,
)
```

Reply:
That would help, we can do it in a new PR.